mojaloop/values.yaml

```diff
@@ -362,7 +374,9 @@ central:
                         "queue.buffering.max.ms": 50,
                         "batch.num.messages": 100,
                         "api.version.request": true,
-                        "dr_cb": true
+                        "dr_cb": true,
+                        "socket.keepalive.enable": true,
+                        "socket.max.fails": 0
```

Comment:
I would suggest also adding the same kafka configs to the "central" & "centralledger" charts:
- "socket.keepalive.enable": true
- "socket.max.fails": 0